sentry-core/src/main/java/io/sentry/core/Sentry.java

```diff
@@ -74,35 +74,35 @@ public static SentryId captureException(Throwable throwable) {
     return getCurrentHub().captureException(throwable);
   }
 
-  public void addBreadcrumb(Breadcrumb breadcrumb) {
+  public static void addBreadcrumb(Breadcrumb breadcrumb) {
```

Comment:
@bruno-garcia please check if all of them need to be `static`, as it is doesn't make sense to me

Reply:
Yeah everything should be static.